RockPaperScissors/RockPaperScissors/MukZziPpa.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  RockPaperScissors - main.swift
+//  Created by Lingo, 우롱차.
+//  Copyright © yagom academy. All rights reserved.
+//
+
+import Foundation
+
+enum MukZziPpaHandType: Int, CaseIterable {
+    case muk = 1
+    case zzi = 2
+    case ppa = 3
+    
+    static var randomPick: Int? {
+        return Self.allCases.randomElement()?.rawValue
+    }
+}
+
+func getPlayerMukZziPpaInput() -> Int? {
+    guard let playerInput = readLine(),
+          let playerIntInput = Int(playerInput)
+    else {
+        return nil
+    }
+    
+    if playerIntInput == Settings.exitCode {
+        return playerIntInput
+    }
+    
+    guard let _ = MukZziPpaHandType(rawValue: playerIntInput) else {
+        return nil
+    }
+
+    return playerIntInput
+}
+
+func printMukZziPpaGameMenu(_ result: GameResult) {
+    switch result {
+    case .playerWin:
+        print(GameDisplayMessage.playerTurnMukzzippaMenu, terminator: "")
+    case .playerLose:
+        print(GameDisplayMessage.computerTurnMukzzippaMenu, terminator: "")
+    default:
+        break
+    }
+}
+
+func startMukZziPpaGame(_ result: GameResult) -> GameResult? {
+    printMukZziPpaGameMenu(result)
+    guard let playerInput = getPlayerMukZziPpaInput() else {
+        print(GameDisplayMessage.invalidPlayerInput)
+        return startMukZziPpaGame(GameResult.playerLose)
+    }
+    if playerInput == Settings.exitCode {
+        print(GameDisplayMessage.gameDidEnd)
+        return nil
+    }
+    guard let computerInput = MukZziPpaHandType.randomPick else {
+        print(GameDisplayMessage.error)
```

Comment:
가위바위보와 묵찌빠에서 사용되는 `손모양`은 동일합니다.
그리고 컴퓨터는 여전히 랜덤한 손을 내잖아요?ㅎㅎㅎ

따라서 여기서 굳이 묵찌빠 만을 위한 손모양 타입(MukZziPpaHandType)을 만들지 마시고
기존 손모양 타입(HandType)을 재사용해보시면 좋겠습니다!

대신 묵찌빠 게임 결과 판단하는 `로직`은 손보셔야 할 거에요!!

```suggestion
    guard let computerInput = HandType.randomPick else {
        print(GameDisplayMessage.error)
```